backend/homework/4/questions/src/main/java/question2/Main.java

```diff
@@ -0,0 +1,31 @@
+package question2;
+
+import com.ganesh.LogMaster;
+
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        TicketReservation mmt = new TicketReservation();
+        String ticketBooked = "Ticket booked: ";
+        String ticketCancelled = "Ticket cancelled: ";
+        String businessClass = "business";
+        String economyClass = "economy";
+        LogMaster.print(ticketBooked + mmt.bookFlight("Ganesh","Setty",21,"male",businessClass,"A1"));
+        LogMaster.print(ticketBooked + mmt.bookFlight("Khushi","Rana",21,"female",businessClass,"A2"));
+        LogMaster.print(ticketBooked + mmt.bookFlight("Sahil","Malik",18,"male",economyClass,"A3"));
+        LogMaster.print(ticketBooked + mmt.bookFlight("Ankush","Rauniyar",22,"male",businessClass,"A4"));
+        LogMaster.print(ticketBooked + mmt.bookFlight("Hema","Nagraj",9,"female",businessClass,"A5"));
+        LogMaster.print(ticketBooked + mmt.bookFlight("Harsh","Verma",22,"male",businessClass,"A6"));
```

Comment:
Lets handle exception scenarios when the booking logic actually breaks. Seprate the logic into a function.